### PR TITLE
DOCS: Add Style Tone Guide + Grammar Docs

### PR DESCRIPTION
**Pull request by HarshCasper**
## Description

This PR:

- Adds the Style Tone Guide
- Adds Grammar Docs for articles, capitalization, abbreviations and acronymns.

### CODE REVIEW

**Review by shubhamkarande13 — APPROVED** [review]

File: docs/DocumentationStyleGuide/style-tone.rst [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+A few guidelines are employed to understand the style and tone. These
+are entirely flexible and are not universal for every context that we
+share amongst ourselves.
+
+Approaches to be considered
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Use Active voice wherever possible.
+-  The style and tone should be friendly and come off as natural.
+-  Making use of transaction words is quite preferred.
```
Comment (aornugent) [inline docs/DocumentationStyleGuide/style-tone.rst:24]: What's a transactional word?

File: docs/DocumentationStyleGuide/style-tone.rst [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+-  Simplify the documentation hierarchy for the readers to easily
```
Comment (aornugent) [inline docs/DocumentationStyleGuide/style-tone.rst:25]: Can you add an example of a simple document hierarchy?

File: docs/DocumentationStyleGuide/style-tone.rst [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+   contributor.
+-  Gender-neutral pronouns must be used for overall inclusivity.
+
+Approaches to be not considered
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Using gender-specific and culture-specific terminologies and figures
+   of speech.
+-  Using biased, unconfirmed or pre-assumed information pieces.
+-  Using pretentious language to convince the reader of a fact or
```
Comment (aornugent) [inline docs/DocumentationStyleGuide/style-tone.rst:40]: This is a good point, but it's not immediately clear what 'pretentious language' - maybe an accessible or humourous example would help?

File: docs/DocumentationStyleGuide/style-tone.rst [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+the projects at moja global forward.
+
+We have some recommended practices involved while writing the
+documentation for these two groups:
+
+For user-based documentation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+While writing user-based documentation, we should consider that many of
+them would not be native English speakers. Hence using strong words,
```
Comment (aornugent) [inline docs/DocumentationStyleGuide/style-tone.rst:62]: Would -> may.

Would and could are generally uncertain terms and should be avoided.

File: docs/DocumentationStyleGuide/style-tone.rst [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+technical jargon, abbreviations would not be very suited for a
+documentation piece. The guidelines in this case are:
+
+-  Maintain a clear and concise knowledge transfer from the
+   documentation to the user.
+-  Use short and crisp sentences that make easy sense to a user.
+-  Use paragraphs to share a single context and point towards one
+   direction only.
```
Comment (aornugent) [inline docs/DocumentationStyleGuide/style-tone.rst:70]: 'towards one direction' is a little ambiguous. Maybes instead give an example of not mixing two points?

File: docs/DocumentationStyleGuide/style-tone.rst [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+-  Consistent vocabulary and phrasing are of the utmost need here.
+-  Actively consider using software or a tool to measure the
+   readability.
```
Comment (aornugent) [inline docs/DocumentationStyleGuide/style-tone.rst:73]: Do you have an example tool?

File: docs/DocumentationStyleGuide/style-tone.rst [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+style as in the user-based documentation. You can assume the technical
+competency of the reader by putting prerequisites and a high-level
+overview of the technology you wish to cover. Some of the guidelines
+are:
+
+-  Use instructive language to convey technical details and steps.
+-  Avoid jumping the bandwagon while documenting technical steps.
+-  Avoid using a conversational tone while documenting code and software
+   references.
+-  Use *the second person* while keeping the emphasis on the reader.
```
Comment (aornugent) [inline docs/DocumentationStyleGuide/style-tone.rst:88]: An example of second vs third person might demonstrate why this works

File: docs/DocumentationStyleGuide/language-grammar/articles.rst [hunk 8]
```diff
@@ -0,0 +1,20 @@
+.. _language--grammar-ii-articles:
+
+Language & Grammar II: Articles
+===============================
+
+Using articles is always preferred in your documentation piece as it
```
Comment (aornugent) [inline docs/DocumentationStyleGuide/language-grammar/articles.rst:6]: This section might need one or more examples so it's more accessible to ESOL readers.

**Review by aornugent — APPROVED** [review]

File: docs/DocumentationStyleGuide/style-tone.rst (hunk 1 above)
Comment (HarshCasper) [reply docs/DocumentationStyleGuide/style-tone.rst:24]: Fixed and added more context.

File: docs/DocumentationStyleGuide/style-tone.rst (hunk 2 above)
Comment (HarshCasper) [reply docs/DocumentationStyleGuide/style-tone.rst:25]: It would be added in the content strategy and we can later link it here.

**Review by iamrajiv — APPROVED** [review]

**Review by chicken-biryani — APPROVED** [review]

**Pull request merged**